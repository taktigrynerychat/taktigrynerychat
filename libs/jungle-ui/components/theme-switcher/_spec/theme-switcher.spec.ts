import { ComponentFixture, TestBed } from '@angular/core/testing';
import { JuiThemeSwitcherComponent } from '@jungle-ui/kit';

describe('JuiThemeSwitcherComponent', () => {
  let component: JuiThemeSwitcherComponent;
  let fixture: ComponentFixture<JuiThemeSwitcherComponent>;

  beforeEach(() => {
    TestBed.configureTestingModule({
      declarations: [JuiThemeSwitcherComponent],
    })
      .overrideTemplate(JuiThemeSwitcherComponent, '<span></span>');
  });

  beforeEach(() => {
    fixture = TestBed.createComponent(JuiThemeSwitcherComponent);
    component = fixture.componentInstance;
  });

  it('Should be created', () => {
    expect(component).toBeTruthy();
  });

});
