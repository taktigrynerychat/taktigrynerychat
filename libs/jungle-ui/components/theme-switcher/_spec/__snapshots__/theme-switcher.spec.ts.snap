// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JuiThemeSwitcherComponent [Function: ngOnChanges()] Should change theme if new value is different and presented in a themes array 1`] = `
Array [
  "data-jui-theme",
  "dark",
]
`;
